packages/core/src/modules/ledger/LedgerApi.ts

```diff
@@ -75,14 +84,33 @@ export class LedgerApi {
 
     const schemaId = generateSchemaId(did, schema.name, schema.version)
 
+    // Generate teh qualified ID
+    // FIXME: How to handle indyLedgers[0]? This should not require selecting the first on eby default
+    const qualifiedIdentifier = getQualifiedIdentifier(
+      this.config.indyLedgers[0].didIndyNamespace,
+      generateSchemaFromTemplate(schemaId, schema)
+    )
+
     // Try find the schema in the wallet
-    const schemaRecord = await this.anonCredsSchemaRepository.findBySchemaId(this.agentContext, schemaId)
-    //  Schema in wallet
-    if (schemaRecord) return schemaRecord.schema
+    const schemaRecord = await this.anonCredsSchemaRepository.findById(this.agentContext, qualifiedIdentifier)
+    // Schema in wallet
+    if (schemaRecord) {
+      // Transform qualified to unqualified
+      // schemaRecord.schema.id = unqualifyIndyDid(schemaRecord.schema.id)
```

Comment:
This shouldn't be commented right?

Reply:
yes indeed

Reply:
Not sure what is going on but my pushed changes are not showing up here